@kiva/kv-tokens/configs/tailwind.config.js

```diff
@@ -124,6 +124,10 @@ module.exports = {
 		}),
 		extend: {
 			typography: kivaTypography.proseOverrides, // prose plugin overrides
+			height: {
+				40: rem(320),
```

Comment:
I think this works well for now. If we end up pulling in the aspect-ratio plugin later on, it'd be nice to use that and keep these misc. height values out of the config.